### PR TITLE
[sdk-127] unsafe assignments changed.

### PR DESCRIPTION
**Pull request by frknbasaran**

### CODE REVIEW

File: lib/countly.js [hunk 1]
```diff
@@ -1725,16 +1725,37 @@
...
+                svgIcon.setAttribute("xmlns", "http://www.w3.org/2000/svg");
+                svgIcon.setAttribute("viewBox", "0 0 496 512");
+                // create path for svg
+                var svgPath = document.createElementNS("http://www.w3.org/2000/svg", "path");
+                svgPath.id = "smileyPathInStickerSvg";
+                svgPath.setAttribute("fill", "white");
+                svgPath.setAttribute("d", "M248 8C111 8 0 119 0 256s111 248 248 248 248-111 248-248S385 8 248 8zm0 448c-110.3 0-200-89.7-200-200S137.7 56 248 56s200 89.7 200 200-89.7 200-200 200zm105.6-151.4c-25.9 8.3-64.4 13.1-105.6 13.1s-79.6-4.8-105.6-13.1c-9.9-3.1-19.4 5.4-17.7 15.3 7.9 47.1 71.3 80 123.3 80s115.3-32.9 123.3-80c1.6-9.8-7.7-18.4-17.7-15.3zM168 240c17.7 0 32-14.3 32-32s-14.3-32-32-32-32 14.3-32 32 14.3 32 32 32zm160 0c17.7 0 32-14.3 32-32s-14.3-32-32-32-32 14.3-32 32 14.3 32 32 32z");
+                // create sticker text wrapper
+                var stickerText = document.createElement("span");
+                stickerText.innerHTML = currentWidget.trigger_button_text;
```
Comment (ar2rsawseen) [inline lib/countly.js]: Codacy Issue found: [Unsafe assignment to innerHTML (no-unsafe-innerhtml/no-unsafe-innerhtml)](https://app.codacy.com/app/ar2rsawseen/countly-sdk-web/pullRequest?prid=4543445)

**Comment by ar2rsawseen** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/countly-sdk-web/countly-sdk-web/pullRequest?prid=4543445&bid=15289058)

File: lib/countly.js (hunk 1 above)
Comment (mt-ronkorving) [reply lib/countly.js]: If `trigger_button_text` doesn't contain HTML, `innerHTML` should be avoided. The span just needs text content. Either `.innerText =` or `.textContent =` will do. I always forget which one is the more cross-browser compatible one :)

Comment (ar2rsawseen) [reply lib/countly.js]: @frknbasaran can you check this too :)

**Comment by ar2rsawseen** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/countly-sdk-web/countly-sdk-web/pullRequest?prid=4543445&bid=15289058)

**Pull request merged**